reflex/vars.py

```diff
@@ -683,6 +792,12 @@ def __ror__(self, other: Var) -> Var:
         Returns:
             A var representing the logical or.
         """
+        other_type = other.type_ if isinstance(other, Var) else type(other)
```

Comment:
Do we have to add this logic for the other operations too? Also, might be worth to see if we can abstract this code out as it's repeated for each function.